dev-libs/argtable/files/argtable-2.13-Fix-undeclared-library-function.patch

```diff
@@ -0,0 +1,28 @@
+From 519609d844f1e5bbf37407de8e43fa2d2be03262 Mon Sep 17 00:00:00 2001
```

Comment:
Have you sent this upstream? If not, why not? If so, include a link :)

Reply:
No. I'm confused about the upstream. Is it https://github.com/argtable/argtable3 or https://github.com/jonathanmarvens/argtable2. If it's the latter, then upstream seems dead.

Reply:
The homepage and src uri are sourceforge?

Reply:
Yes. The last release was January 10th, 2011.

Reply:
so why are we talking about github?